src/test/java/org/broadinstitute/hellbender/tools/PrintReadsIntegrationTest.java

```diff
@@ -151,4 +152,29 @@ public void testReadThatConsumesNoReferenceBases() throws IOException {
         //Make sure we print the read, ie not lose it.
         SamAssertionUtils.assertSamsEqual(outFile, zeroRefBasesReadBam);
     }
+
+    @Test
+    public void testNoConflictRG() throws IOException {
+        final File inFile = new File(TEST_DATA_DIR, "print_reads_withPG.sam");
+        final File outFile = BaseTest.createTempFile("testNoConflictRG", ".sam");
+        final String[] args = new String[] {
+                "--input" , inFile.getAbsolutePath(),
+                "--addOutputSAMProgramRecord",
+                "--output", outFile.getAbsolutePath()
+        };
+        // Make sure no exception is thrown given an input containing a read that consumes no reference bases
```

Comment:
Does this file actually have a read that doesn't consume reference bases?  Which one is it?  It looks like they all have a section of Match except for the unmapped one.